content/blog/getting-started-with-pac/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Getting Started With Pac"
+date: 2020-02-11
+meta_desc: "Pulumi CrossGuard validates infrastructure before deployment to prevent data breaches caused by misconfigured services, "
+meta_image: crossguard.svg
```

Comment:
Does Twitter support displaying .svg?

Reply:
Also, if we're not using the `meta.png` file, it should be deleted from this directory.

Reply:
It has on previous tweets, meta.png deleted.